src/lib/cache.js

```diff
@@ -168,13 +125,23 @@ export default {
 
   delete: key =>
     new Promise((resolve, reject) =>
-      client.del(key, (err, response) => {
+      client.del(key, (err) => {
         if (err) return reject(err)
-        resolve(response)
       })
     ),
 
   isAvailable: () => {
-    return client.ping()
+    return new Promise((resolve, reject) => {
+      client.stats((err, resp) => {
+        if (err) {
+          error(err)
+          reject(err)
+        } else {
+          if (resp){
+            resolve(resp)
+          }
```

Comment:
This looks like there’s the potential for the promise never resolving or rejecting. That’s not a problem for the promise, but seeing as the caller isn’t handling this possibility in any way it could mean there’s HTTP health requests that never get responded to.

Reply:
Will remove the nested `if` - it's redundant if either `err` or `resp` is always provided, but true that it doesn't guard against the edge case where neither is.